src/Framework/Blocks/BlockModel.php

```diff
@@ -73,6 +73,18 @@ public function getAttributes(): array
         return $this->attributes;
     }
 
+    /**
+     * @unreleased
+     */
+    public function setAttribute(string $name, $value): BlockModel
```

Comment:
Can you please update the `TestBlockModel` unit test to include this.

Reply:
Resolved baf7036aaa610288e16b4f37de0f017316ecbefe